airbyte-integrations/connectors/source-mixpanel/metadata.yaml

```diff
@@ -33,6 +33,13 @@ data:
   releaseStage: generally_available
   releases:
     breakingChanges:
+      3.0.0:
+        message:
+          In this release, state for CohortMembers is changed to per partition format.
+          Key for CohortMembers stream was changed to new unique key based on 
+          distinct_id and cohort_id fields since previous key was not unique and 
+          didn't support possibility for user be in a few different cohorts
+        upgradeDeadline: "2024-05-22"
```

Comment:
please set upgradeDeadline to 2024-06-03

Reply:
updated